UI/Web/src/app/manga-reader/_models/reader-enums.ts

```diff
@@ -1,7 +1,8 @@
 export enum FITTING_OPTION {
     HEIGHT = 'full-height',
     WIDTH = 'full-width',
-    ORIGINAL = 'original'
+    ORIGINAL = 'original',
+    MANUAL = 'manual'
```

Comment:
I don't think we need this at all. I think we can just have a slider that applies an arbitrary width with the user is in WIDTH fitting.